roles/api/tasks/hasura-install.yml

```diff
@@ -199,7 +199,7 @@
 - name:  wait for hasura port to become available
   wait_for:
     port: "{{ api_port }}"
-    host: "{{ api_ip_address }}"
+    host: "{{ api_hostname }}"
```

Comment:
this should be localhost imho (http should never be access via network)
but we might have an issue if install host <> backend server (but it works for devsrv installs, so then again no issue